core/domain/user_services.py

```diff
@@ -406,7 +406,10 @@ def record_user_started_state_editor_tutorial(user_id):
     _save_user_settings(user_settings)
 
 
-def update_email_preferences(user_id, can_receive_email_updates):
+def update_email_preferences(
```

Comment:
You see when new user is added this function is called with only first 2 arguments. Since at time of sign up we are not going to give user a choice for receiving this type of emails, I thought that 3rd argument should ensure that receiving should be set to default value. Thus adding 3rd argument as default arg maintains older implementation of this function. If it is explicit then we need to pass some value to it which will be default value. So rather than modifying function call I thought why not change it so it will work for older calls as well as for new features both.

Now what you are saying in this comment. No value won't change like that. The calls made for this function will be changed during the milestone 2. Generally what I was thinking that, whichever property user wants to change, we will simply transfer values for both properties from front end to backend. After that we will call this function using 3 arguments. So you see changes will be reflected in Model. But that will be done in milestone 2 because if we do it now in this commit then we are yet to implement frontend facility for user to be able to change value but we are changing backend facility and it will result in errors.

Now the other point you said that we use a different function for this. But then we will have 2 different functions to change same model's different properties and will not it lead to some confusion ?
I think we should go implement it the way I did because it does not break any current workings of oppia and in milestone 2 we are going to modify function call from 2 args to 3 args.

So how does that sound ?